features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -440,11 +440,16 @@
     "/api/identity/config-mgt/v1.0/",
     "/api/users/v2/me/webauthn",
     "/api/",
-    "/user-portal/"
+    "/user-portal/",
+    "/authenticationendpoint/",
+    "/accountrecoveryendpoint/"
   ],
   "tenant_context.rewrite.servlets": [
     "/identity/(.*)",
-    "(.*)/identity/metadata/saml2"
+    "(.*)/identity/metadata/saml2",
+    "/commonauth(.*)",
+    "/logincontext(.*)",
+    "/oidc/"
```

Comment:
Shall we put the changes of default.json and the identity.xml.j2 in a seperate PR? @malithie WDYT?

Reply:
yes please and link that one here